runtime/crab/src/lib.rs

```diff
@@ -261,6 +263,13 @@ frame_support::construct_runtime! {
 		EVM: darwinia_evm::{Pallet, Call, Storage, Config, Event<T>} = 39,
 		Ethereum: dvm_ethereum::{Pallet, Call, Storage, Config, Event, ValidateUnsigned} = 40,
 		// DynamicFee: dvm_dynamic_fee::{Pallet, Call, Storage, Inherent} = 42,
+
+		// S2S bridge.
+		BridgeDispatch: pallet_bridge_dispatch::<Instance1>::{Pallet, Event<T>} = 46,
```

Comment:
I think changing to `BridgeDarwiniaDispatch` would be better, can avoid unnecessary runtime maintenance work at least. There might be a `BridgeXXXDispatch` pallet in the future. @HackFisher @AurevoirXavier

Reply:
I am not so sure about this after seeing discusses in slack. 😨 Or just leave it as is.

Reply:
> I think changing to `BridgeDarwiniaDispatch` would be better, can avoid unnecessary runtime maintenance work at least. There might be a `BridgeXXXDispatch` pallet in the future. @HackFisher @AurevoirXavier

Sound good for me, make sense to some extend. I saw some source chain related account types configured.